src/main/java/frc/robot/subsystems/HopperSubsystem.java

```diff
@@ -6,33 +6,37 @@
 
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import frc.robot.Constants;
-import com.revrobotics.CANSparkMax;
-import com.revrobotics.CANSparkMaxLowLevel;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import frc.robot.common.hardware.MotorController;
 
 /** Add your docs here. */
 public class HopperSubsystem extends SubsystemBase {
   // Put methods for controlling this subsystem
   // here. Call these from Commands.
-  private CANSparkMax m_hopperController;
-  
+  private MotorController[] mHopperController = new MotorController[1];
```

Comment:
No reason to use an array here. Maybe if you expect to have more motors added eventually, but I don't think we do.